tests/unit/plugins/action/test_es_data_inputs_network.py

```diff
@@ -370,15 +369,49 @@ def test_es_data_inputs_network_merged_idempotent(self, conn, monkeypatch):
         self._plugin._connection._shell = MagicMock()
 
         # patch get operation
-        get_response = RESPONSE_PAYLOAD["tcp_cooked"]
+        get_response = {
+            "entry": [
+                {
+                    "name": "default:8100",
+                    "content": {
+                        "connection_host": "ip",
+                        "disabled": False,
+                        "host": "$decideOnStartup",
+                        "restrictToHost": "default",
+                    },
+                },
+            ],
+        }
 
         def get_by_path(self, path):
             return get_response
 
+        def create_update(
+            self,
+            rest_path,
+            data=None,
+            mock=None,
+            mock_data=None,
+        ):
+            return get_response
+
         monkeypatch.setattr(SplunkRequest, "get_by_path", get_by_path)
+        monkeypatch.setattr(SplunkRequest, "create_update", create_update)
 
         # tcp_cooked
-        get_response = RESPONSE_PAYLOAD["tcp_cooked"]
+        get_response = {
+            "entry": [
```

Comment:
Instead of rewriting this here, can we not do `RESPONSE_PAYLOAD["tcp_cooked"]["entry"]`?

Reply:
Will do, thanks